contrib/bdii/htcondor-ce-provider

```diff
@@ -199,6 +205,8 @@ def main():
             sys.stderr.write("Unable to locate CE schedd on %s" % ce_host)
             continue
 
+        signal.signal(signal.SIGALRM, handler)
+        signal.alarm(10)
```

Comment:
Can we make the timeout window based off an HTCondor configuration value with a default of 10s? If you don't know how to do this, I can submit a PR

Reply:
Will add this option